### ISSUE TITLE
differentiate unlocked theorems from axioms in theorem list

### ISSUE THREAD

**Issue by mcol** [hhu-adam/NNG4#70]
For example, `pow_zero` shows this text: "pow_zero a : a ^ 0 = 1 is one of the two axioms defining exponentiation in this game." If `pow_zero` is indeed an axiom, perhaps it should have a different colour or icon, to differentiate it from the unlocked theorems.

**Comment by joneugster** [comment]
This is an interesting point. I guess for game play it doesn't really matter what are axioms and what are theorems.

Making them colourful might draw too much attention on them when the attention should rather go to the newly introduced theorems (which are colourful in the next update)

**Comment by mcol** [comment]
I agree, colour would not work well. Perhaps as a tooltip? But feel free to close if this suggestion goes beyond or against the objectives of the game.

**Comment by joneugster** [comment]
That's a good idea. At the place in `lean4game` where the signature is autogenerated, one could also check whether the statement is an axiom.

**Comment by KatherineCox** [comment]
I agree this would be helpful.

Context: I’m self-studying some math/CS and stumbled across the Natural Numbers Game. I’ve seen some proofs before, (I’m currently working through Susanna Epp’s Discrete Math textbook) but this is my first exposure to Peano arithmetic.

While working through the first couple of Worlds, I got pretty confused about which things we were treating as true “because we say so” (axioms and definitions) vs. which things we were proving based on those axioms and definitions. It doesn’t help that (a) they all live in the same place (i.e. `add_zero` and `zero_add` are not noticeably distinguishable from each other under Theorems), and (b) the way they’re introduced is basically the same:
1. Here’s a new theorem and what it does.
2. Here’s a goal. Use this new theorem to prove the goal.

As a newbie, there’s a lot that I’m trying to hold in my head, and it’s easy to lose the forest for the trees. I think any signposting that you can provide to keep track of where I started and where I’m going would be helpful as I try to follow the big picture. (The home screen with the different “Worlds” is great for this, by the way.)
